source/routing/rendering-a-template.md

```diff
@@ -25,16 +23,7 @@ export default Ember.Route.extend({
 });
 ```
 
-If you want to use a different controller than the route handler's
-controller, pass the controller's name in the `controller` option:
-
-```app/routes/post.js
-export default Ember.Route.extend({
-  renderTemplate() {
-    this.render({ controller: 'favoritePost' });
-  }
-});
-```
+## Named Outlets
```

Comment:
I am fairly convinced that we don't want to encourage named outlets for anything. As an alternative, we need to write examples like a basic sidebar and modal using component + service.